front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -103,6 +101,17 @@ export function AgentMessage({
     { index: number; document: RetrievalDocumentType | WebsearchResultType }[]
   >([]);
 
+  useEffect(() => {
```

Comment:
Do we expect to have several visualizations in one run? If so, this is fuego 🔥!

Reply:
Yeah with this setup we support it. I think it's quite niche still, but nice to have it working (in some cases it can be useful)